omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -165,6 +165,12 @@
       </Directory>
     </Directory>
 
+    <Property Id="DATADOGYAMLEXISTS">
+      <DirectorySearch Id="DatadogYamlSearchDirId" Path="[CommonAppDataFolder]\Datadog" Depth="12" AssignToProperty="yes">
```

Comment:
@julien-lebot we have ability to store configuration in other than ProgramData\Datadog directory? I actually did try to use APPLICATIONDATADIRECTORY but it was empty because the property was set later and I was not sure if I would be able to set my property before required dialogs. If we always store in ProgramData\Datadog than it appears to be more complicated to realize.

Reply:
By what means?

Reply:
... also the property DATADOGYAMLEXISTS is set and matter technically ONLY when a fresh installation is commenced. Previous installation technically does not exist and the CUSTOM path to the file cannot be discovered. Am I missing something.